cadwyn/structure/versions.py

```diff
@@ -446,7 +446,10 @@ async def _convert_endpoint_response_to_version(
             path,
             method,
         )
-        if isinstance(response_or_response_body, FastapiResponse):
+        if isinstance(response_or_response_body, FastapiResponse) and response_info.body:
```

Comment:
Please, let's move the second part of this `if` down in the same manner as in `main`

Reply:
fixed